plugins/idea/pluginImpl/src/main/java/com/imaginea/kodebeagle/impl/util/ScalaImportsUtil.java

```diff
@@ -29,9 +30,12 @@
 import com.intellij.psi.util.PsiTreeUtil;
 import java.util.Map;
 import java.util.Set;
-import org.jetbrains.plugins.scala.lang.psi.impl.ScalaFileImpl;
+// import org.jetbrains.plugins.scala.lang.psi.impl.ScalaFileImpl;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)